src/components/ha-climate-control.html

```diff
@@ -67,38 +70,47 @@
       },
     };
   }
+  temperatureStateInFlux(inFlux) {
+    this.$.target_temperature.classList.toggle('in-flux', inFlux);
+  }
   incrementValue() {
-    var newval = this.value + this.step;
-    this.last_changed = Date.now();
+    const newval = this.value + this.step;
     if (newval <= this.max) {
-      this.value = newval;
+      // If no initial target_temp
+      // this forces control to start
+      // from the min configured instead of 0
+      if (newval <= this.min) {
+        this.value = this.min;
+      } else {
+        this.last_changed = Date.now();
+        this.temperatureStateInFlux(true);
+        this.value = newval;
+      }
     } else {
       this.value = this.max;
     }
   }
   decrementValue() {
-    var newval = this.value - this.step;
-    this.last_changed = Date.now();
+    const newval = this.value - this.step;
     if (newval >= this.min) {
```

Comment:
I think the following case is unhandled:
min=10
value=10.5
step=1

the result is less than min, but setting value=min still changes value.
Everything would still work, but the 2s time won't get restarted.

Reply:
I improved the detection of value changed to put temperature color correctly and trigger last_changed trigger